ONSdigital/takeon-ui
# added condition to clear any pre-filled period dates before submittin…

…g values

# Description
Added condition to clear any pre-filled dates before validating.

# How to test?
Run RSI bdd tests against the spp2109 env to check for this change running only few tests like popmrz should be ok and also able to check for the numeric question tactical fix story where the tests not been used to submit any numeric values for form 6 and 7 should pass and not stop validation process

` behave features/rsi/rsi_validation_popmrz.feature -D browser=headless
`
